### DISCUSSION TITLE
Maggie Appleton

### DISCUSSION THREAD

**Discussion by hipstersmoothie** [devtools-fm/devtools.fm#291] [Announcements]
- https://twitter.com/Mappletons
- https://maggieappleton.com/
- https://elicit.com/

## Introduction

- designer at Elicit
- host of the future of coding in London
- advocate for digital gardens

### Digital Gardening

- [x] You've written a lot about digital gardening. Could you explain the concept for our listeners who may not have heard of it?
- [x] How has it helped you with forming ideas? What are some resources you’d suggest to people trying to get into it?
- [x] You wrote _A Brief History & Ethos of the Digital Garden_ back in 2020. Since then it seems the concept has had much broader adoption, especially with the rise of tools for thought as a mainstream theme. How do you feel about its current adoption and what would you like to see going forward?
- [x] You’re a great visual storyteller and make graphics that are excellent at communicating concepts. What is your process for creating these visual metaphors?

### LLMs and their impact on how we think and work

- [x] In a world where computers can craft an essay from a prompt, why should we keep writing? Likewise when we can get summaries of any text, what keeps us reading?
- [x] What is Elicit? 
- [x] How do you think we should go about designing AI in our apps?

### Future of coding

- [x] You've been hosting the future of coding in London for a little while now. What motivated you to host that event and what draws you to the space?
- [x] What tech has you the most excited about the future of code?
- [x] There's overlap of people from the tools for thought space who now are talking a lot about local first. From cozy web to cozy apps. Do you have thoughts on why this overlap exists?

**Comment by just-be-dev** [comment]
Tooltips:
- https://github.com/atomicojs/atomico
- https://superstate.io/

**Comment by hipstersmoothie** [comment]
https://github.com/stanford-oval/storm
https://next-view-transitions.vercel.app/

**Comment by hipstersmoothie** [comment]
https://www.youtube.com/watch?v=wjZofJX0v4M
